rotkehlchen/db/dbhandler.py

```diff
@@ -1970,6 +1970,9 @@ def edit_exchange(
             try:
                 exchange_name = new_name if new_name is not None else name
                 self.set_binance_pairs(exchange_name, binance_markets, location)
+                # Also delete used query ranges to allow fetching missing trades
+                # from the possible new pairs
+                self.delete_used_query_range_for_exchange(location)
```

Comment:
Shouldn't it only delete trades used query range? Since deposit/withdrawals should not be affected?

Reply:
Yes, with the trades range is enough